src/components/Calculator.js

```diff
@@ -1,56 +1,50 @@
 import React, { Component } from 'react';
 import '../App.css';
+import calculate from '../logic/calculate';
 
 export default class Calculator extends Component {
```

Comment:
- [ ] Your calculator works perfectly well but can you please implement the use of life cycle methods as this was one of the learning objectives of this project

Reply:
Great! This is looks perfect! I think the learning objectives here have all been attained.👍💯